week-2/F-logical-operators/exercise2.js

```diff
@@ -5,7 +5,22 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(a) {
+  if (a < 0) return true;
+  else return false;
+}
+function isBetween5and10(a) {
+  if (a > 5 && a <= 10) return true;
+  else return false;
+}
+function isLessThan8Characters(name) {
+  if (name.length < 8) return true;
+  else return false;
+}
+function startsWithD(name) {
+  if (name.startsWith("D")) return true;
+  else return false;
+}
```

Comment:
As above, any time you find yourself writing `if (something) return true; else return false;` then this can be simplified to `return something`.

Reply:
i did it a lot :)